dockerfile creats incomplete container

curling the git repo, only downloads the source, not the git repo, so we cannot 
```
git submodule update --init
```
and it builds a container with the following error when running the tests 

```

@:~/Coding/ffig_proj/rest-api$ docker build -t ffig/web-base:latest .
@:~/Coding/ffig_proj/rest-api$ docker exec stoic_lalande bash -c "/home/ffig-master/scripts/build.py -t"
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ffig-master/build
[ 20%] Built target Tree_c
[ 40%] Built target Asset_c
[ 60%] Built target Shape_c
[ 66%] Building CXX object tests/src/CMakeFiles/TestCppCircle.dir/TestCppCircle.cpp.o
/home/ffig-master/tests/src/TestCppCircle.cpp:2:21: fatal error: catch.hpp: No such file or directory
compilation terminated.
make[2]: *** [tests/src/CMakeFiles/TestCppCircle.dir/TestCppCircle.cpp.o] Error 1
make[1]: *** [tests/src/CMakeFiles/TestCppCircle.dir/all] Error 2
tests/src/CMakeFiles/TestCppCircle.dir/build.make:62: recipe for target 'tests/src/CMakeFiles/TestCppCircle.dir/TestCppCircle.cpp.o' failed
CMakeFiles/Makefile2:200: recipe for target 'tests/src/CMakeFiles/TestCppCircle.dir/all' failed
Makefile:94: recipe for target 'all' failed
make: *** [all] Error 2
Traceback (most recent call last):
  File "/home/ffig-master/scripts/build.py", line 78, in <module>
    main()
  File "/home/ffig-master/scripts/build.py", line 67, in main
    'cmake --build ./{}'.format(args.out_dir).split(), cwd=src_dir)
  File "/usr/lib/python2.7/subprocess.py", line 541, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmake', '--build', './build']' returned non-zero exit status 2

```

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,12 +14,11 @@ RUN pip2 install --upgrade pip && \
 
 # Install the ffig codebase. Use `cd` here to avoid several WORKDIR layers.
 RUN cd /home && \
-    curl -LOk https://github.com/FFIG/ffig/archive/master.zip && \
-    unzip master.zip && \
-    rm -f master.zip
+    git clone -b master --recurse-submodules https://github.com/FFIG/ffig.git && \
+    touch __init__.py ffig/__init__.py
 
 # Copy in the content of this repository
 COPY . /home/flask/
-
 WORKDIR /home/flask/
 
+CMD ["python", "ffig_explorer.py"]

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,11 +14,14 @@ RUN pip2 install --upgrade pip && \
 
 # Install the ffig codebase. Use `cd` here to avoid several WORKDIR layers.
 RUN cd /home && \
+    # clone and init the git submodules
     git clone -b master --recurse-submodules https://github.com/FFIG/ffig.git && \
-    touch __init__.py ffig/__init__.py
+    touch ffig/__init__.py
+
+ENV PYTHONPATH $PYTHONPATH:/home/ffig/ffig/
 
 # Copy in the content of this repository
 COPY . /home/flask/
 WORKDIR /home/flask/
 
-CMD ["python", "ffig_explorer.py"]
+#CMD ["python", "ffig_explorer.py"]

